spec/models/results/csv/generator_spec.rb

```diff
@@ -231,12 +241,12 @@
     let(:relation) do
       # Simulate some conditions like we'd get from a search.
       Response.for_mission(get_mission).joins(:form)
-        .where("(((responses.reviewed = 'n')) AND ((forms.name ILIKE '%Sample%')))")
+        .where("(((responses.reviewed = 'n')) OR ((forms.name ILIKE '%Sample%')))")
     end
 
     before do
       Timecop.freeze(Time.zone.parse("2015-11-20 12:30 UTC")) do
-        create_response(form: form1, mission: missions[0], answer_values: ["foo"])
+        create_response(form: form1, mission: missions[0], answer_values: ["foo"], reviewed: true)
```

Comment:
I appreciate that you're trying to exercise the reviewed code throughout this spec. It's a very thorough energy you're bringing! But you really only need to do it in perhaps one place so that you've proven that the values coming out in the CSV really correspond to the ones you've set (e.g. they're not just 'true' everytime).

This is generally something to keep in mind in your work— ask yourself: "what is the least number (and type) of tests I need to write here to properly cover the code I'm adding". Tests are great but they are also code that needs to be written AND maintained, so keeping them to a minimum (without sacrificing coverage) is important.

So with that in mind I don't think you need to disturb as much of this file or e.g. change the SQL fragment above. I think you could go with the factory default of `false` except in one of the above where you have a mixture. That would increase the simplicity of your changes here, which is generally a good thing!